dspace-api/src/main/resources/org/dspace/storage/rdbms/sqlmigration/postgres/V7.2_2022.12.06__index_action_resource_policy.sql

```diff
@@ -0,0 +1,9 @@
+--
```

Comment:
The Flyway migration process doesn't apply this migration. The version should be updated, it should be V7.5_2022.12.06__index_action_resource_policy.sql ?